librz/core/cmd_analysis.c

```diff
@@ -7909,16 +7909,16 @@ static void rz_core_analysis_info(RzCore *core, const char *input) {
 	}
 }
 
-static void cmd_analysis_aad(RzCore *core, const char *input) {
-	RzListIter *iter;
-	RzAnalysisXRef *xref;
-	RzList *list = rz_analysis_xrefs_get_from(core->analysis, UT64_MAX);
-	rz_list_foreach (list, iter, xref) {
-		if (xref->type == RZ_ANALYSIS_REF_TYPE_DATA && rz_io_is_valid_offset(core->io, xref->to, false)) {
-			rz_core_analysis_fcn(core, xref->from, xref->to, RZ_ANALYSIS_REF_TYPE_NULL, 1);
-		}
+bool aad_cb(RzAnalysisXRef *xref, void *user) {
```

Comment:
I suggest to give better names than `aad_cb` and `cmd_analysis_aad` while you are on it.

Reply:
I'll rename `aad_cb` but leave `cmd_analysis_aad` for others since these handlers anyway will be ported to the newshell soon.